src/tests/specs/iteration.spec.ts

```diff
@@ -23,6 +23,7 @@ describe('Iteration test', () => {
     let month = await planner.iteration.getMonth();
     let year = await planner.iteration.getYear();
     await planner.iteration.clickCreateIteration();
+    expect(await planner.iteration.checkVisibility()).toBeFalsy();
```

Comment:
@divyanshiGupta if you are checking visiblity of an element always handle the exception otherwise test will fail because first step is to find the element and then perform an action.